cache_invalidator/Makefile

```diff
@@ -0,0 +1,12 @@
+CC?=gcc 
```

Comment:
This repo appears to follow a convention of putting whitespace around the operators in `Makefile`s. (make allows this, shell scripts do not).